virttest/utils_hotplug.py

```diff
@@ -483,3 +486,44 @@ def vcpuhotunplug_unsupport_str():
         return "not currently supported"
     else:
         return ""
+
+
+def create_mem_xml(params):
```

Comment:
we can have some minimal default args and based on those inputs we can override param values or fallback if not provided?, for example, a simple case of xml with memory size.

Reply:
This is the complete set for memory hotplug xml. I use all these parameters in hotplug testing. All parameters has default values,  Where ever it is missing I will add it up.  As it is complete set, it can be used generically.